roles/kubernetes/preinstall/defaults/main.yml

```diff
@@ -1,6 +1,9 @@
 ---
 run_gitinfos: false
 
+# Skip installation of the EPEL repository
+skip_epel: false
```

Comment:
It would be better to use consistent var names for enabling or skipping a particular component. Can you rename this var to `epel_enabled` with a default of true?